### DISCUSSION TITLE
Get rid of manually curated release notes

### DISCUSSION THREAD

**Discussion by ricardoV94** [pymc-devs/pymc3#5890] [Ideas]
We could just rely on Github [automatically generated release notes](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#about-automatically-generated-release-notes), which is what we do in Aesara and Aeppl. This seems to work pretty well and generate [useful release notes](https://github.com/aesara-devs/aesara/releases)

This way we don't have to keep remembering to modify the Release Note in every PR (yay), and we won't forget to mention any PR (double yay). 

This may also save quite some CI Runs, as in most cases the Release Note are only updated after the PR tests pass for the first time.

The release notes can be configured automatically (e.g., remove the autoupdate pre-commit config)  and/or tweaked manually as well when needed.

**Comment by ferrine** [comment]
Agree, this is gold

**Comment by michaelosthege** [comment]
I like that feature too, but I think it will be super important to make releases much more frequently. 
Otherwise it'll be super hard to reformat the release notes into something that's easy to comprehend and on-point.

We can also start using the autogeneration feature __before__ stopping to curate the `RELEASE-NOTES.md`.

**Reply by ricardoV94** [reply]
Yeah let's try it for the next releases
